src/sentry/monitors/tasks/detect_broken_monitor_envs.py

```diff
@@ -33,6 +34,9 @@
 # The number of days a monitor env has to be failing to qualify as broken
 NUM_DAYS_BROKEN_PERIOD = 14
 
+# The number of days a monitor env has to be broken to be auto-muted
+NUM_DAYS_MUTED_PERIOD = 14
```

Comment:
Maybe we can make the comment more clear here that this is AFTER it has moved into the broken state.